.env

```diff
@@ -0,0 +1 @@
+CIVIC_API_KEY=AIzaSyBYRFTlLS33NiXJ_OwXNRL-XNrqqNknkJU
```

Comment:
should this be gitignored and we all set it in our local env, or are we just not worrying about that for a while?

Reply:
Yes definitely. I was planning on just destroying the key once we had a solution for .env variables. There's a package that I had installed called react-native-dotenv that turns the .env file into a named export. so you can `import {API_KEY} from 'react-native-dotenv'` But I wasn't sure how to get it into CircleCI and it was breaking tests on there.